components/blog/Author.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="pb-3">
+    <strong>{{ name }}</strong>
+    <br>
+    <span v-if="organisation">
+      {{ organisation }}
+    </span>
+    <SmartLink
+      v-if="url"
+      :destination="url"
+    >
+      {{ url }}
+    </SmartLink>
+  </div>
+</template>
+
+<script>
+  import SmartLink from '../generic/SmartLink';
+
+  export default {
+    name: 'Author',
```

Comment:
[Component names should always be multi-word.](https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential)